src/Files.App/Services/Settings/GeneralSettingsService.cs

```diff
@@ -263,6 +263,12 @@ public FileNameConflictResolveOptionType ConflictsResolveOption
 			set => Set(value);
 		}
 
+		public bool DisplaySettingsButtonOnSidebar
+		{
+			get => Get(false);
```

Comment:
```suggestion
			get => Get(true);
```
The UI for this can be displayed under experimental.